src/Executor/Executor.php

```diff
@@ -258,14 +258,14 @@ private static function buildExecutionContext(
                 $rawVariableValues ?: []
             );
 
-            if ($coercedVariableValues['errors']) {
-                $errors = array_merge($errors, $coercedVariableValues['errors']);
-            } else {
+            if (empty($coercedVariableValues['errors'])) {
                 $variableValues = $coercedVariableValues['coerced'];
+            } else {
+                $errors = array_merge($errors, $coercedVariableValues['errors']);
             }
         }
 
-        if ($errors) {
+        if (! empty($errors)) {
```

Comment:
Is there a real/technical benefit to this?

- I always prefer to read the "positive" form: "if something is…"
  than
- the negated form "if something is not …"

Reply:
I don't bother much, but just curious - is it phpstan requirement and we'll have to stick with it or just your preference?

Reply:
PHPStan preference is not to use `empty()` at all but use `count()` (cuz `empty()` is less strict but IMO for arrays it's just fine). That's why I added it as excluded rule (otherwise it would force to change all empty()ies to count()s).

But here the check is that you have to use boolean comparison in IF statement, that is eg. `if ($value !== null)` instead of `if ($value)`

Reply:
@vladar what about this? Have I addressed everything?

Reply:
@simPod I am travelling at the moment, so not always available

Reply:
@vladar  sure, thanks for letting me know. Enjoy ur travels ;)